Staking Page Reorg (#3787)
w3f/polkadot-wiki: docs/learn/learn-staking.md

are made public to all nominators, and a nominator, in turn, submits a list of up to
{{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominations" defaultValue={16}/> :polkadot }}
{{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}
candidates that it supports, and the network will automatically distribute the stake among
validators in an even manner so that the economic security is maximized. In the next era, a certain
number of validators having the most {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} backing
get elected and become active. For more information about the election algorithm go to
[this](learn-phragmen.md) page on the wiki or
[this](https://research.web3.foundation/en/latest/polkadot/NPoS/1.%20Overview.html?highlight=proportional%20justified%20representation#)
research article. As a nominator, a minimum of
{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}
{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }}

> is required to submit an intention to nominate. Note that in NPoS the stake of both nominators and

Reviewer: ...is required to submit an intention to nominate, **which can be thought of as registering to be a nominator.**

(trying to make clear what "intention" is.

Reviewer 2: Specified

validators can be slashed. For an in-depth review of NPoS see
[this](https://research.web3.foundation/en/latest/polkadot/NPoS/index.html) research article.

:::caution Minimum Nomination to Receive Staking Rewards

Although the minimum nomination intent is
{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }},
it does not guarantee staking rewards. The nominated amount has to be greater than
[minimum active nomination](learn-nominator.md#minimum-active-nomination-to-receive-staking-rewards),
which is a dynamic value that can be much higher than

> {{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }}.

Reviewer: This dynamic value depends on the amount of {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} being staked, in addition to the selected nominations.

Reviewer 2: Added

[session keys](https://research.web3.foundation/en/latest/polkadot/keys/index.html?highlight=session%20keys),
produce new block candidates in [BABE](learn-consensus.md/#block-production-babe), vote and come to
consensus in [GRANDPA](learn-consensus.md/#finality-gadget-grandpa), validate the state transition
function of parachains, and possibly some other responsibilities regarding data availability and
[XCM](learn-cross-consensus.md). For more information, you can take a look at the
[validator docs](learn-validator.md) to understand what you need to do as a validator. If you want
to become a validator you can consult
[this](../maintain/maintain-guides-how-to-validate-polkadot.md) guide.

**Nominators.** Nominators have far fewer responsibilities than validators. These include selecting validators and
monitoring their performance, keeping an eye on changing commission rates (a validator can change
commission at any time), and general health monitoring of their validators' accounts. Thus, while
not being completely set-it-and-forget-it, a nominator's experience is relatively hands-off compared

> to that of a validator, and even more with [nominator pools](./learn-nomination-pools.md). For more

Reviewer: change nominator pools to nomination pools.

Reviewer 2: Done

stored on the device (PC, phone) and it is always exposed to the internet for potential hacks and
scams. It is good practice to deposit rewards on the stash account or to send them to another
account on a cold wallet.

:::

![staking](../assets/staking/stash-controller.png)

This hierarchy of separate keys for stash and controller accounts was designed to add a layer of
protection to nominators and validator operators. The more often one exposes and uses a private key,
the higher its vulnerability for hacks or scams. So, if one uses a key for multiple roles on a
blockchain network, it is likely that the account can get compromised. Note that the damage linked
to stolen private keys is different depending on the type of account derivation. In the case of soft
derivation, all derived accounts are compromised. More information about account derivation can be
found [here](../learn/learn-accounts.md/#derivation-paths).

:::info

For Ledger users staking directly on Ledger Live, currently, there is no option to use separate
stash and controller accounts. That is if you stake on Ledger Live your stash account will be your

> controller too.

Reviewer: Yes this is an issue. We might want to add that Ledger is now supported in Talisman extension. Users can import their Ledger accounts in there and use them as a stash and controller.

Ledger still doesn't support key extrinsics like identity though (2022, shocking!) which is why I personally hold off from using Ledger.

Reviewer 2: Added

a validator. For more information, you can take a look at the [nominator guide](learn-nominator.md)
to understand your responsibilities as a nominator.

If you want to become a nominator, see
[this](../maintain/maintain-guides-how-to-nominate-polkadot.md) guide. If you are a beginner and
would like to securely stake your tokens using Polkadot JS Apps, refer to [this](https://support.polkadot.network/support/solutions/articles/65000168057-how-do-i-stake-nominate-on-polkadot-) support article. {{ kusama: The tutorial presented in the support article is demonstrated on Polkadot, but the procedure is the same for Kusama :kusama }}

### Selection of Validators

The task of choosing validators is not simple, as it should take into account nominator reward and
risk preferences. Ideally one aims to maximize the reward-to-risk ratio by maximizing rewards and minimizing risks, with sometimes having to
compromise between the two, as minimizing risks might decrease rewards as well. Nominators should pay attention, especially to six criteria when nominating validators (not in order of importance):

- recent history of the era points earned across eras

> - validator's (own) stake (shows skin in the game)

Reviewer 3: We refer to Validator's own stake as Self Stake https://wiki.polkadot.network/docs/maintain-guides-validator-community#self-stake You could also link to this doc here

Reviewer 2: Alright then I need to change the figure. I took the terminology from the staking tab on the UI. I wanted anyway to add the figure for the staking proxy and the updated figure for the stash-controller.

Reviewer 3: Got it. In that case, let us leave it as is. Should we may be add a snapshot of Polkadot JS UI along with the figure and pointing to the respective parameters?

Reviewer 2: New Figures added, updated Figure risk-rewards 👍

> 

Reviewer 4: Do we have resources to have somewhat better grapghics? If not, can we at least use the Polkadot font? a proper background color that's Polkadot themed?

Reviewer 2: Hi @kianenigma I am using google docs to create the graphics and it does not allow to install the Polkadot font. I will need to see what are other options with W3F. I initially produced the images with Polkadot branding colour as background but I switched to a white background using the branding colour for the font. It seemed "cleaner" to me. But I can revert it. What do you mean exactly with "better graphics"? Is the overall presentation of the diagrams not good, or just the font and the background colour?

Reviewer 3: Let me check with the design team on this. We were already thinking about revamping the graphics on the Wiki

Reviewer 2: I spoke to Iggy and he told me to use Inter font for the figures. Polkadot unbounded font is only used for titles. I updated all figures accordingly.

import RPC from "./../../components/RPC-Connection"

Here you will learn about what staking is, why it is important and how it works on
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}.

## Introduction to Proof-of-Stake (PoS)

Blockchain networks use [consensus](learn-consensus.md/#why-do-we-need-consensus) mechanisms to finalize
blocks on the chain. Consensus is the process of agreeing on something, in this case, the progression
of the blockchain or how blocks are added to the chain. Consensus consists of two actions:

> - `block production`, i.e. the way multiple blocks candidates are produced, and 

Reviewer 4: I would not use code formatting for what is not code.

Reviewer 2: Fixed throughout the page

- `block finality`, i.e. the way only one block out of many candidates is selected and added to the canonical chain (see [this](learn-consensus.md/#probabilistic-vs-provable-finality) article for more information about finality).

> Proof-of-Work (PoW) and Proof-of-Stake (PoS) are well-known mechanisms used to reach consensus in a secure and trustless way on public blockchains, where there are many participants who do not know each other (and probably never will). In PoW, network security relies on the fact that the miners who are responsible for adding blocks to the chain must compete to solve difficult mathematic puzzles to add blocks - a solution that has been criticized for the wastage of energy. For doing this work, miners are typically rewarded with tokens. In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the

Reviewer 4: one paragraph for each would be a cleaner description, and I generally think both can be a bit more concise (but that subjective).

Reviewer 2: Suggestion accpeted

They are instead pre-selected to produce the blocks based on the stake backing them.
Token holders can lock funds on the chain and for doing so, they are getting `staking rewards`. There is
thus an economic incentive for token holders to become active participants who contribute to the
security and economic stability of the network. PoS networks in general are therefore more inclusive
than PoW networks, as participants do not need to have either technical knowledge about
blockchain technology or experience in running mining equipment. 

PoS ensures that everybody participating in the staking process has
"skin in the game" and thus can be held accountable. In case of misbehavior, participants in the
staking process can be punished or `slashed`, and depending on the gravity of the situation, their
stake can be partly or fully confiscated by the network. It is not in a staker's economic interest to orchestrate an attack and risk losing tokens. Any rational actor staking on the network would want to get rewarded, and the PoS network rewards good behavior and punishes bad behavior.

> ## Nominated Proof-of-Stake Overview

Reviewer: Is it worth having a section here to distinguish the difference between nominators, validators and pools? Perhaps it would be good to say Pools is "built" on top of NPoS to provide a very low barrier to entry to staking, without sacrificing Polkadot's strict security model.

{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} implements [Nominated Proof-of-Stake (NPoS)](learn-consensus.md/#nominated-proof-of-stake),
a relatively novel and sophisticated mechanism to select the validators who are allowed to participate in 
its [consensus](learn-consensus.md) protocol. The NPoS encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
holders to participate as nominators. NPoS encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}
holders to participate as `nominators`. 

Nominators may back up to
{{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominations" defaultValue={16}/> :polkadot }}
{{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}

> validators as trusted validator candidates, and the network will automatically distribute the stake among validators in

Reviewer 4: autoamtically, such that the economic security is maximized. 

And ideally "economic security" should be explained in:

```
Similar to the miners in PoW networks, PoS networks have...
```

an even manner. Also, in NPoS the stake of both nominators and validators can be slashed.  For an in-depth review of NPoS see
[this](https://research.web3.foundation/en/latest/polkadot/NPoS/index.html) research article.

### Nominating Validators

> The action of nominating consists of a) locking or

Reviewer: The "lock", "bond" and "stake" terms tend to be used interchangeably. Maybe simplify to:

Nominating on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} requires 2 actions:

 a) Locking tokens on-chain.
 b) Selecting a set of validators, to whom these locked tokens will automatically be allocated to. 

How many tokens you lock up is completely up to you - as are the validators you wish to select.

_Locking tokens is also known as **bonding**.  You can also refer to your locked tokens as your **bonded** tokens, or **staked** tokens. Likewise, selecting validators is also known as **backing** or **nominating** validators. These terms are used interchangeably by the community._

From now on locked tokens will be referred to as **bonded** tokens.

Once the previous 2 steps are completed and you are nominating, your bonded tokens could be allocated to one or more of your selected validators, and this happens _every time the active **validator set** changes_. 

This validator set is updated every **era** on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}. On Polkadot, an era is 1 day.

Unlike other staking systems, Polkadot automatically chooses which of your selected validators will be backed by your bonded tokens. Selecting a group of validators increases your chances of consistently backing at least one who is active. This results in your bonded tokens being allocated to validators more often, which means more network security and more rewards.

This is in stark contrast to other staking systems that only allow you to back one validator; if that validator is not active, you as a staker will also not be. 

Polkadot's nomination model solves this.

> bonding tokens (stake) on the chain, and b) nominating a set of validator candidates to whom the stake will

Reviewer 4: make a proper markdown list please?

Reviewer 2: Done

from election theory to game theory to discrete optimization, to develop an efficient validator
selection process that offers fair representation and security, thus avoiding uneven power and
influence among validators. The election algorithms used by {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} are based on the Proportional Justified Representation (PJR) methods like [Phragmen](learn-phragmen.md). For more information about PJR methods visit [this](https://research.web3.foundation/en/latest/polkadot/NPoS/1.%20Overview.html?highlight=proportional%20justified%20representation#) research article.

### Eras and Sessions

The stake from nominators is used to increase the number of tokens held by such candidates, increasing their chance of being selected by the election algorithm for block
production during a specific `era`. An era is a period of {{ polkadot: 24 hours :polkadot }}{{ kusama: 6 hours :kusama }} during
which an `active set` of validators is producing blocks and performing other actions on chain. This
means that not all validators are in the active set and such set changes between eras. Each era is

> divided into 6 epochs or `sessions` during which validators are assigned as block producers to

Reviewer 4: in the spirit of conciseness, maybe we can skip epoch and session details here. 

Who is the audience here? What I am reading is not super advance, but it is also not geared towards a non-crypto-native end user. Is that a fair assumption?

Reviewer 2: @kianenigma I introduced the terms here, and defined them appropriately because they are used later in the page and I believe is important for the reader to know what these things are in general, and they must know it as soon as possible before the explanations become complex. Let me know if you agree with this.

To know more about era points, and how and on which basis they are distributed visit the
[dedicated page](../maintain/maintain-guides-validator-payout.md). Distribution of the rewards is pro-rata to all stakers after the validator's commission is deducted.

### Skin in the game when Staking

The security of PoS networks depends on the amount of staked tokens. To successfully attack the network, a malicious actor would need to accrue a large number of tokens or would need different participants to collude and act maliciously. If there is an attack in the case of NPoS, both the validator(s) and nominators will be
`slashed` resulting in their stake being partially or fully confiscated by the network and then deposited to the treasury. There is little interest for a rational network participant to act in a harmful way because NPoS ensures that all participants can be
held accountable for their bad actions. In NPoS, validators are paid equal rewards regardless of the amount of stake backing them, thus avoiding large payouts to few large validators which might lead to centralization.

## Being a nominator

### Tasks and Responsibilities of a Nominator

> Since validator slots are limited, most of those who wish to stake their DOT and contribute to the

Reviewer 4: DOT should be DOT / KSM?

Reviewer 2: yep, thanks. This is fixed now.

Reviewer 3: We can also conditionally render this doc for Polkadot and Kusama

produce new block candidates in [BABE](learn-consensus.md/#block-production-babe), vote and come to
consensus in [GRANDPA](learn-consensus.md/#finality-gadget-grandpa), validate the state transition
function of parachains, and possibly some other responsibilities regarding data availability and
[XCM](learn-cross-consensus.md). For more information, you can take a look at the
[validator docs](learn-validator.md) to understand what you need to do as a validator. If you want
to become a validator you can consult [this](../maintain/maintain-guides-how-to-validate-polkadot.md) guide.

Nominators have far fewer responsibilities than validators. These include selecting validators and
monitoring their performance, keeping an eye on changing commission rates (a validator can change
commission at any time), and general health monitoring of their validators' accounts. Thus, while not
being completely set-it-and-forget-it, a nominator's experience is relatively hands-off compared to that of
a validator. For more information, you can take a look at the [nominator guide](learn-nominator.md)

> to understand your responsibilities as a nominator.

Reviewer 4: Here you should mention that one step more passive than nomination is joining a pool.

Reviewer 2: Added

If you want to become a nominator, see
[this](../maintain/maintain-guides-how-to-nominate-polkadot.md) guide. If you are a beginner and

> would like to securely stake your tokens using Polkadot JS Apps, refer to [this](https://support.polkadot.network/support/solutions/articles/65000168057-how-do-i-stake-nominate-on-polkadot-) support article. {{ kusama: The tutorial presented in the support article is demonstrated on Polkadot, but the procedure is the same for Kusama :kusama }}

Reviewer 4: do we have an equivalent in the staking dashboard?

Reviewer 3: @filippoweb3 Please link to this article as well https://support.polkadot.network/support/solutions/articles/65000182133-how-to-use-the-staking-dashboard-staking-your-dot

Reviewer 2: Added

### Selection of Validators

> The task of choosing validators is not simple, as it should take into account nominator reward and

Reviewer 4: have we considered justifying the rendered text? it probably makes it a lot more readable.

{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }}
is required to submit an intention to nominate.

:::caution Minimum Nomination to Receive Staking Rewards

Although the minimum nomination intent is
{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }},
it does not guarantee staking rewards. The nominated amount has to be greater than
[minimum active nomination](learn-nominator.md#minimum-active-nomination-to-receive-staking-rewards),
which is a dynamic value that can be much higher than
{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }}.

:::

> ### Bags List

Reviewer 4: We should move this as low/far as possible, because ideally users don't have to deal with this. Can it be moved further down, or to a separate page?

Reviewer 2: I agree. Moved at the end of the Section "Staking System Overview"

  account. The earned rewards can be bonded (locked) immediately for staking on your stash account,
  which would effectively compound the rewards you receive over time. You could also choose to have
  them deposited to your controller account or a different account as a free (transferable) balance.
  If you are a validator, it can also be used to set your [session keys](learn-keys.md#session-keys). Controller
  accounts only need sufficient funds to pay for the transaction fees.

:::warning

Never leave a high balance on a controller account which are usually "hot" as their
private key is stored on the device (PC, phone) and it is always exposed to the internet for potential hacks and scams. It is good practice to deposit rewards on the stash account or to send them to another
account on a cold wallet.

:::

![staking](../assets/NPoS/staking-keys_stash_controller.png)

This hierarchy of separate key types for stash and controller accounts was designed so that nominators and validator operators can
protect themselves much better than in systems with only one key. As a rule, the more often one uses

> a private key the higher its visibility and thus the chance it can be stolen. So, if one uses a key

Reviewer 3: ```suggestion
This hierarchy of separate keys for stash and controller accounts was designed to add a layer of protection to nominators and validator operators. The more often one exposes and uses a private key, the higher its vulnerability for hacks or scams. So, if one uses a key
```

for multiple roles on a blockchain network, security can be easily compromised as the likelihood one

> uses that key often is high. Note that the damage linked to stolen private keys is different

Reviewer 3: ```suggestion
for multiple roles on a blockchain network, it is likely that the account can get compromised. Note that the damage linked to stolen private keys is different
```

depending on the type of account derivation. In the case of soft derivation, all derived accounts are
compromised. More information about account derivation can be found
[here](../learn/learn-accounts.md/#derivation-paths).

:::info

For ledger users staking directly on Ledger Live, there is currently only one option to use one

> account as both stash and controller.

Reviewer 3: ```suggestion
For ledger users staking directly on Ledger Live, currently there is no option to use separate stash and controller accounts.
```

:::

### Staking proxies

{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} is built using

> [substrate](https://substrate.io/), a modular system to efficiently build blockchains without having necessarily deep knowledge about blockchain technology. Within each module or `pallet`, one can `call` different

Reviewer 3: ```suggestion
[substrate](https://substrate.io/), a modular system to efficiently build blockchains. Within each module or `pallet`, one can `call` different
```

functions that have similar logic. For example, the staking pallet contains all functionalities
related to staking such as bonding or unbonding funds. The combined information of pallets and calls
constitutes an `extrinsic`, i.e. a transaction that is executed from outside the chain but that

> triggers an event within the chain. Continuing with the staking example, within the staking pallet

Reviewer 3: ```suggestion
triggers an event on the chain. Continuing with the staking example, within the staking pallet
```

> one can bond funds and nominate some validators. The signature of such

Reviewer 3: Are you sure about bonding funds and nominating validators triggers rewards payouts? I think the idea you are trying to convey here will need rephrasing.

Reviewer 2: Re-phrased

extrinsic might trigger a reward payout at the end of an era; this is an event inside the chain. This way of having
transactions categorized within pallets and functionalities makes it possible to create accounts
having special permissions also called **proxy accounts**.

In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special

> accounts to which one can delegate signatures to calls from specific pallets. There is thus the

Reviewer 3: ```suggestion
In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special
accounts which can sign extrinsic calls made to specific pallets on behalf of the proxied account. There is thus the
```

possibility to create staking proxy accounts that can be used to sign only calls from staking,

> session and utility pallets. This makes the stash account even more isolated than using a controller

Reviewer 3: ```suggestion
possibility to create staking proxy accounts that can be used to sign only extrinsic calls to staking,
session and utility pallets. This makes the stash account even more isolated than using a controller
```

> account since now one can bond / unbond / bond more funds using the staking proxy account. However,

Reviewer 3: ```suggestion
account since one can bond / unbond / bond more funds using the staking proxy account. However,
```

it is important to remember that actions on proxy accounts are limited, and in the case of staking
proxy accounts, calls from the balance pallet cannot be signed. This means that it is not possible to

> send funds from a staking proxy. To do that one needs to remove that account as a staking proxy.

Reviewer 3: ```suggestion
it is important to remember that actions that can be performed by the proxy accounts are limited, and in the case of staking
proxy, extrinsic calls to the balances pallet cannot be signed. This means it is not possible to do balance transfers
on the proxied account through a staking proxy.
```

nomination intents, of which, at most {{ polkadot: <RPC network="polkadot" path="query.electionProviderMultiPhase.maxElectingVoters" defaultValue={22500}/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.electionProviderMultiPhase.maxElectingVoters" defaultValue={20000}/> :kusama }}
come out as the electing nominators. Check
[Staking Election Stages](learn-nominator.md#staking-election-stages) section for more info.

:::caution Minimum active nomination threshold to earn rewards is dynamic

Once again, submitting a nomination intent does not guarantee staking rewards. The stake of the top
{{ polkadot: <RPC network="polkadot" path="query.electionProviderMultiPhase.maxElectingVoters" defaultValue={22500}/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.electionProviderMultiPhase.maxElectingVoters" defaultValue={20000}/>  :kusama }} nominators is applied to the
validators in the active set. To avail of staking rewards, ensure that the number of tokens bonded is
higher than the minimum active nomination. For more information, check the
[nominator guide](learn-nominator.md)

:::

> The solution to the election of nominators has to meet certain requirements, such as maximizing the

Reviewer 3: "The solution of the election of nominators" is ambiguous. The election solution comprises of both nominators and validators. It is primarily for electing validators for the active set

```suggestion
The "election solution" which is a connected graph between nominators and validators with the stake as edge weights, has to meet certain requirements, such as maximizing the
```

amount of stake to nominate validators and distributing the stake backing validators as evenly as
possible. The objectives of this election mechanism are to maximize the security of the network, and
achieve fair representation of the nominators. If you want to know more about how NPoS works (e.g.
election, running time complexity, etc.), please read
[here](http://research.web3.foundation/en/latest/polkadot/NPoS.html).

### Rewards distribution

:::info

The general rule for rewards across validator pools is that two validator pools get paid essentially
the same amount of tokens for equal work, i.e. they are not paid proportional to the stakes in
each pool. There is a probabilistic component to staking rewards in the form of
[era points](../maintain/maintain-guides-validator-payout.md##era-points) and
[tips](learn-transaction-fees.md#fee-calculation) but these should average out over time.

:::

> Validator pools are paid the same regardless of stake level. Pools with less stake will generally pay more to nominators

Reviewer 3: ```suggestion
Validators are paid the same regardless of stake backing them. Validators with less stake will generally pay more to nominators
```

> per-token than pools with more stake. This gives nominators an economic incentive to gradually shift their preferences to lower-staked validators that gain a sufficient amount of reputation. A consequence of this is that the stake across validator pools will be as evenly distributed as possible, to avoid a concentration of power among a few validators. In the long term, validator pools will have similar levels of stake, with the stake

Reviewer 3: ```suggestion
per-token than the ones with more stake. This gives nominators an economic incentive to gradually shift their preferences to lower-staked validators that gain a sufficient amount of reputation. A consequence of this is that the stake across validators will be as evenly distributed as possible which avoids concentration of power among a few validators. In the long term, validators will have similar levels of stake, with the stake
```

> being higher for higher reputation validators (meaning that a nominator that is willing to risk more
> by backing a validator with a low reputation will get paid more).
> 

Reviewer 3: ```suggestion
being higher for validators with higher reputation. A nominator who is willing to risk more
by backing a validator with a lower reputation will get paid more, provided there are no slashing events.

```

Before distributing rewards to nominators, validators can create a cut of the reward (a commission) that is not shared with the nominators.
This cut is a percentage of the block reward, not an absolute value. After the commission gets

> deducted, the remaining portion is based on their staked value and split between the validator and
> all of the nominators who have voted for this validator.

Reviewer 3: ```suggestion
deducted, the remaining portion is distributed pro-rata based on their staked value and split between the validator and
all of the nominators whose stake has backed this validator.
```

For example, assume the block reward for a validator is 10 DOT. A validator may specify
`validator_commission = 50%`, in which case the validator would receive 5 DOT. The remaining 5 DOT
would then be split between the validator and their nominators based on the proportion of stake each

> nominator had. Note that validators can put up their own stake, and for this calculation, their
> stake acts just as if they were another nominator.

Reviewer 3: ```suggestion
nominator had. Note that for this calculation, validator's self-stake acts just as if they were another nominator.
```

> Within a validator pool, a percentage of the reward goes thus to pay the validator's

Reviewer 3: ```suggestion
Thus, a percentage of the reward goes thus to pay the validator's
```

mitigated by the validator operator running separate validators; each validator is considered its
own entity for purposes of slashing, just as they are for staking rewards.

> In rare instances, a nominator may be actively nominating several validators in a single era. In

Reviewer: Or with very large bonds such as parachain liquid staking accounts (Acala's LDOT nominator typically has 7-12 active nominations per era).

Reviewer 2: Interesting. Added in the text.

can pay rewards. In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} this limit is currently {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :polkadot }}{{ kusama: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :kusama }}, although this can be
modified via runtime upgrade. A validator with more than {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :polkadot }}{{ kusama: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :kusama }} nominators is
_oversubscribed_. When payouts occur, only the top {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :polkadot }} nominators as measured by the amount of stake allocated to that validator will receive rewards. All other nominators
are essentially "wasting" their stake - they used their nomination to elect that validator to the
active stake, but receive no rewards in exchange for doing so.

> Note that the network slashes a validator slot for a misbehavior (e.g. validator

Reviewer 3: ```suggestion
Note that the network slashes a validator for a misbehavior (e.g. validator
```

offline, equivocation, etc.) the slashed amount is a fixed percentage (and not a fixed amount),
which means that validator pools with more stake get slashed more DOT. Again, this is done to
provide nominators with an economic incentive to shift their preferences and back less popular
validators whom they consider to be trustworthy.

> Also note that each validator candidate is free to name their desired commission

Reviewer 3: ```suggestion
Also, note that each validator candidate is free to name their desired commission
```

fee (as a percentage of rewards) to cover operational costs. Since validator pools are paid the
same, pools with lower commission fees pay more to nominators than pools with higher fees. Thus,
each validator can choose between increasing their fees to earn more, or decreasing their fees to
attract more nominators and increase their chances of being elected. In the long term, we expect
that all validators will need to be cost-efficient to remain competitive, and that validators with
higher reputation will be able to charge slightly higher commission fees (which is fair).

### Claiming rewards

{{ kusama: Note that Kusama runs approximately 4x as fast as Polkadot, except for block production times.
Polkadot will also produce blocks at approximately six-second intervals. :kusama }}

> Rewards are recorded per session (approximately {{ polkadot: 4 hours :polkadot }}{{ kusama: one hour :kusama }}) and
> calculated per era (approximately {{ polkadot: 24 hours :polkadot }}{{ kusama: 6 hours :kusama }}). Thus,
> rewards will be calculated {{ polkadot: once per day :polkadot }}{{ kusama: 4 times per day :kusama }}. Rewards are calculated based on era points, which have a probabilistic component. In other words,

Reviewer 3: This was from an earlier PR

```suggestion
Rewards are calculated per era (approximately six hours on Kusama and twenty-four hours on Polkadot). These rewards are calculated based on era points, which have a probabilistic component. In other words,
```

there may be slight differences in your rewards from era to era, and even amongst validators in the
active set at the same time. These variations should cancel out over a long enough timeline. See the
page on [Validator Payout Guide](../maintain/maintain-guides-validator-payout.md).

> In order to be paid your staking rewards, someone must claim them for each validator that you
> nominate. Staking rewards are kept available for 84 eras, which is approximately {{ polkadot: 84 days :polkadot }}{{ kusama: 21 days :kusama }}. For more information on why this is so, see the page on [simple payouts](learn-simple-payouts.md).

Reviewer 3: > In order to be paid your staking rewards, someone must claim them for each validator that you
nominate.

A bit ambiguous

```suggestion
The distribution of staking rewards to the nominators is not automatic and needs to be triggered by someone. Typically the validators take care of this, but anyone can permissionlessly trigger rewards payout for all the nominators whose stake has backed a specific validator in the active set of that era. Staking rewards are kept available for 84 eras, which is approximately {{ polkadot: 84 days :polkadot }}{{ kusama: 21 days :kusama }}. For more information on why this is so, see the page on [simple payouts](learn-simple-payouts.md).
```

Reviewer 2: I agree thanks for the suggestion

:::info Payouts

Payouts are unclaimed rewards waiting to be paid out to both validators and nominators. If you go to the Staking payouts page on
[Polkadot-JS](https://polkadot.js.org/apps/#/staking), you will see a list of all validators
that you have nominated in the past 84 eras and for which you have not yet received a payout. The payout page is visible only to stakers.

:::

Each validator as well as their nominators have the option to trigger the payout for all unclaimed eras. Note that this will pay everyone
who was nominating that validator during those eras. Therefore, you may not
see anything in this tab, yet still have received a payout if somebody (generally, but not
necessarily, another nominator or the validator operator) has triggered the payout for that
validator for that era.

> :::warning Time limit

Reviewer 3: ```suggestion
:::warning Time limit to claim staking rewards
```

If nobody claims your staking rewards within 84 eras, then you will not be able to claim them and they will be lost. Additionally, if the validator unbonds all their own stake, any pending payouts will also be lost. Since unbonding takes {{ polkadot: <RPC network="polkadot" path="query.staking.bondingDuration" defaultValue={28}/> days :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.bondingDuration" defaultValue={7}/> days :kusama }}, nominators should check if they have pending payouts at least this often.

:::

> Rewards can be directed to the same account used to sign the payout (controller), to the stash account (and either

Reviewer 3: ```suggestion
Rewards can be directed to the same account used to sign the payout (controller), or to the stash account (and either
```

large, Polkadot will trigger a new validator election to get a full set. Disabled validators will
need to resubmit their intention to validate and re-garner support from nominators.

For more on chilling, see the "[How to Chill][]" page on this wiki.

> ## Why and why not staking?

Reviewer: Is "Why and why not to stake?" more natural?

Reviewer 2: Changed

> the number of validators has not been determined yet, but should only be limited by the bandwidth

Reviewer: Is this outdated?

Reviewer 2: probably. Removed.
